pkg/codegen/go/gen.go

```diff
@@ -1333,7 +1333,7 @@ func (pkg *pkgContext) genOutputType(w io.Writer, baseName, elementType string,
 	// Generate 'ToOuput(context.Context) pulumix.Output[T]' method
 	// to satisfy pulumix.Input[T].
 	goPackageInfo := goPackageInfo(pkg.pkg)
-	if goPackageInfo.Generics == GenericsSettingSideBySide {
+	if goPackageInfo.Generics == GenericsSettingSideBySide || goPackageInfo.Generics == GenericsSettingGenericsOnly {
```

Comment:
This is the only place where `side-by-side` didn't also imply `generics-only` which caused the issue.